conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -67,7 +67,7 @@ touch /home/conda/feedstock_root/build_artifacts/conda-forge-build-done
 EOF
 
 set +x
-echo "BINSTAR_TOKEN=${BINSTAR_TOKEN}" > "${FEEDSTOCK_ROOT}/env"
+echo "BINSTAR_TOKEN=${BINSTAR_TOKEN}" > "${FEEDSTOCK_ROOT}/.circleci/env"
```

Comment:
Noted in the other PR, but what about adding this to a directory that isn't mounted in the container?

Reply:
Does it really matter?

Reply:
It defeats the purpose a bit if we add it to a directory that is mounted in the container and it's trivial to move it outside. Why not do it?

Reply:
Even if it's not mounted, it's trivial to create a file right? `echo $BINSTAR_TOKEN > ~feedstock_root/env2`

Reply:
It doesn't really defeat the purpose. The env file isn't available in the container because it's deleted before the container runs (mounting volumes is a link, not a copy).